Data/SampleData.cs

```diff
@@ -1,33 +1,36 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Identity;
+﻿using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.DependencyInjection;
 
-namespace PhoneEdit.Data
+namespace PhoneEdit.Data;
+
+public class SampleData
 {
-    public static class SampleData
+    private static IConfiguration _configuration;
+
+    public SampleData(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+    // Creates default app's admin
+    public static async Task CreateDefaultUser(IServiceProvider serviceProvider)
     {
-        // Creates default app's admin
-        public static async Task CreateDefaultUser(IServiceProvider serviceProvider)
+        var userManager = serviceProvider.GetRequiredService<UserManager<IdentityUser>>();
+        var configuration = serviceProvider.GetRequiredService<IConfiguration>();
+        
+        var defaultUsername = configuration.GetValue<string>("DefaultUsername");
+        var defaultPassword = configuration.GetValue<string>("DefaultPassword");
+
+        var userCheck = await userManager.Users.AnyAsync();
+        if (!userCheck)
         {
-            var userManager = serviceProvider.GetRequiredService<UserManager<IdentityUser>>();
-  
-            
-            var userCheck = await userManager.Users.AnyAsync();
-            if (!userCheck)
+            var adminUser = new IdentityUser()
             {
-                var adminUser = new IdentityUser()
-                {
-                    UserName = "mail@example.com",
-                    Email = "mail@example.com",
-                    EmailConfirmed = true
-                };
-                
-                await userManager.CreateAsync(adminUser, "123456");
-            }
+                UserName = Environment.GetEnvironmentVariable("USERNAME") ?? defaultUsername,
```

Comment:
since you already using IConfiguration it's not required to explicitly read the env variable (IConfiguration already does it)

to make it even more consistent you could use IOptions<T> with a separate section for a default user configuration

Reply:
If I understood correctly, you wanted to see something like this: [Commit](https://github.com/rumkit/PhoneEdit/pull/8/commits/6268581d6d9b4e800dc35350290c59636f43caa8)

Reply:
exactly!